projects/components/src/table/cells/data-renderers/code/code-table-cell-renderer.component.ts

```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { TableCellRenderer } from '../../table-cell-renderer';
+import { TableCellRendererBase } from '../../table-cell-renderer-base';
+import { CoreTableCellParserType } from '../../types/core-table-cell-parser-type';
+import { CoreTableCellRendererType } from '../../types/core-table-cell-renderer-type';
+import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
+
+@Component({
+  selector: 'ht-code-table-cell-renderer',
+  styleUrls: ['./code-table-cell-renderer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="code-cell" [htTooltip]="this.value | htDisplayString">
```

Comment:
Is this code or just version? We might have to do more to support code view right?

Reply:
Version was just the example. It's code like backticks are `code` in markdown - not supposed to be multiline support in a table cell, although I guess we could add that if needed.